### ISSUE TITLE
Off Peak & percent_offpeak have no values, all in kwh_peak

### ISSUE THREAD

**Issue by anothermort** [hypercrypt/octopus-db#15]
Is the idea to eventually have kWh_offpeak and percent_offpeak calculated and in teh table below?

period | kwh_total | kwh_peak | kWh_offpeak | percent_offpeak | cost | pperkwh | tariffs |  
-- | -- | -- | -- | -- | -- | -- | -- | --
2023-01-07 | 12.241 | 12.241 | 0.000 | 0.0 | 2.568276 | 20.98 | E-2R-AVRO-VAR-21-09-26-A
2023-01-06 | 7.050 | 7.050 | 0.000 | 0.0 | 1.662580 | 23.58 | E-2R-AVRO-VAR-21-09-26-A
2023-01-05 | 7.465 | 7.465 | 0.000 | 0.0 | 1.745567 | 23.38 | E-2R-AVRO-VAR-21-09-26-A
2023-01-04 | 8.651 | 8.651 | 0.000 | 0.0 | 1.987898 | 22.98 | E-2R-AVRO-VAR-21-09-26-A
2023-01-03 | 7.085 | 7.085 | 0.000 | 0.0 | 1.784341 | 25.18 | E-2R-AVRO-VAR-21-09-26-A
2023-01-02 | 7.633 | 7.633 | 0.000 | 0.0 | 1.682231 | 22.04 | E-2R-AVRO-VAR-21-09-26-A

Excellent work thanks for all the work.
Working on a grafana dashboard to share, but not up to your programming skills.

Thanks

Adam

**Comment by hypercrypt** [comment]
Absolutely. Because I built this for myself until recently and I've been on EV tariffs for the majority of time the off-peak threshold is hard-coded to 10p at the moment, a number that has been sufficient for me until I got round to fetching it properly. What is your off-peak rate at the moment?

**Comment by anothermort** [comment]
Good to hear, I patiently await future updates.

My current off-peak rate is 16.087p.

Thanks for the great software.

**Comment by NiccyB** [comment]
Just in case there was a plan to move to automatic calculation of the off-peak threshold, can I request that setting it remains a manual process? I would rather have the control of deciding what the value is for myself - when (if) I move to Agile again, I may need to change the off-peak value as and when the tariff trends up or down.

**Comment by hypercrypt** [comment]
I see four different cases for "off-peak"

Standard - no off peak except for promotions 

E7 - off peak is the low rate

Go/Intelligent - almost the same as E7 but based on half hourly prices

Agile - here I see a few options

- base it on the cheapest x slots each day
- long term average, lowest x% of slots
- hard code a value in `config.php`

**Comment by NiccyB** [comment]
For the Agile options, can I add "store the value in a database table"?
